packages/ihe-gateway/server/Channels/CareQuality Document Exchange - Outbound/XCA ITI-39 Processor/sourceConnector-transformer-step-0-Set stats.js

```diff
@@ -1,6 +1,6 @@
 // Store internal message id
 channelMap.put('MSG_ID', msg.id.toString());
 channelMap.put('CUSTOMER_ID', msg.cxId.toString());
-
+channelMap.put('REQUEST_TIME', DateUtil.getCurrentDate("yyyy-MM-dd'T'hh:mm:ss.SSSZ"));
```

Comment:
Let's create a function to get the current time? Should we have the need to make any changes, it will make it easier and less likely to miss a spot.